types/index.d.ts

```diff
@@ -161,6 +161,10 @@ declare namespace math {
     isHexDigit(c: string): boolean
   }
 
+  interface NodeCtor {
+    new (): MathNodeCommon
```

Comment:
https://github.com/josdejong/mathjs/pull/2592/files#diff-093ad82a25aee498b11febf1cdcb6546e4d223ffcb49ed69cc275ac27ce0ccceR165

This was the issue 🤦🏻  I had apparently not realized that there is no `Node` type in Mathjs. I'm not sure why my TS is behaving differently than for @egziko, but for me TS was inferring the DOM `Node` type and so not raising an error for an undefined type
<img width="528" alt="image" src="https://user-images.githubusercontent.com/64985/175781865-dabf748e-ed9a-4438-9841-52dfb90067ce.png">